portfolio/src/main/webapp/script.js

```diff
@@ -88,30 +88,52 @@ function getComments() {
 getComments();
 
 /**
- * Checks login status of user, links to login/logout page, hides comment form
- * if user is logged out.
+ * Fetches login status data from /login and updates comment-form div in DOM
+ *     depending on login status. Depending on status, DOM is updated to hide
+ *     the form and prompt the user to login or out.
  */
-function checkLoginStatus() {
+function updateCommentForm() {
```

Comment:
Since client code is accessible in the browser, anyone can modify your JS or send a direct request to your submit comment endpoint without being logged in. You should also verify that the user is logged in that endpoint.